data/Archlinux.yaml

```diff
@@ -1,4 +1,5 @@
 ---
 mcollective_choria::package_dependencies:
   "ruby-nats-pure": "present"
+  "mcollective": "present"
```

Comment:
@bastelfreak is this still right? a package called mcollective to install there? This is moving from here https://github.com/choria-io/puppet-mcollective/pull/283

Reply:
I think we can delete this. the mcollective package works only in combination with the puppet 5 agent.